index.html

```diff
@@ -3,9 +3,35 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="./style.css">
+    <script src="https://kit.fontawesome.com/c72a7bd198.js" crossorigin="anonymous"></script>
+    <title>Carmax - Browse used cars and new cars online</title>
 </head>
 <body>
-    
+    <header>
+        <nav>
+          <ul>
```

Comment:
@Afrocentriccynic Instead of one `ul`, you should make use of two `uls`. One for the list on the right, another for the list on the left. Separate them using flexboxes.